oak_functions_service/src/lookup.rs

```diff
@@ -25,11 +25,14 @@ use hashbrown::HashMap;
 use log::{info, Level};
 use spinning_top::{RwSpinlock, Spinlock};
 
-use crate::logger::OakLogger;
+use crate::{logger::OakLogger, lookup_htbl::LookupHtbl};
+
+// This is used to pass data to the lookup data module for tests.
+pub type Data = HashMap<Bytes, Bytes>;
```

Comment:
I don't think there's much value in having this public type at all by this point.

The only place left over where it's used after your changes is `LookupDataManager::for_test`. Either just use the HashMap type directly there, or arguably we should change it to take `IntoIterator<Item = (Bytes, Bytes)>` much like `extend_next_lookup_data` does.

Reply:
I agree with this and the following comment about making Data private.  However, it does impact multiple files, and will grow this PR.  Data should be passed as Vec<(Bytes, Bytes)> IMO.  I'll make that change and we can see how it looks.

Reply:
Done.